pkg/controllers/nodeclaim/termination/controller.go

```diff
@@ -71,7 +71,7 @@ func (c *Controller) Reconcile(ctx context.Context, n *v1beta1.NodeClaim) (recon
 
 //nolint:gocyclo
 func (c *Controller) finalize(ctx context.Context, nodeClaim *v1beta1.NodeClaim) (reconcile.Result, error) {
-	ctx = log.IntoContext(ctx, log.FromContext(ctx).WithValues("node", nodeClaim.Status.NodeName, "provider-id", nodeClaim.Status.ProviderID))
+	ctx = log.IntoContext(ctx, log.FromContext(ctx).WithValues("Node", klog.KRef("", nodeClaim.Status.NodeName), "provider-id", nodeClaim.Status.ProviderID))
```

Comment:
Two questions here:
1) Is there a reason we're moving to capitalize resource names? Don't have a problem with it, just curious.
2) Should other names be consistent, e.g. `ProviderID` instead of `provider-id`

Reply:
K8s objects that are injected by controller-runtime are capitalized. I'd like to see us be consistent with K8s objects in the way that we return them and print them the args.